uaclient/entitlements/tests/test_esm.py

```diff
@@ -73,7 +73,7 @@ def fake_exists(path):
                     entitlement.name
                 ),
                 "http://{}".format(entitlement.name.upper()),
-                "TOKEN",
+                "%s-token" % entitlement.name,
```

Comment:
A nit, but `.format` here would be more consistent with the rest of the codebase.